src/config/config.rs

```diff
@@ -241,6 +244,8 @@ pub struct Field {
   #[serde(default, skip_serializing_if = "is_default")]
   pub modify: Option<Modify>,
   #[serde(default, skip_serializing_if = "is_default")]
+  pub omit: Option<Omit>,
```

Comment:
```suggestion
  pub omit: Option<bool>,
```
I don't think we need a struct in this case.